Add support for defmt. Add support for dumping register contents at the end of init function.

matoushybl opened the pull request.

richardeoin reviewed (approved):
Thanks for the PR! Looks good to me

Would you consider adding some text to the usage section in the README to help people find the defmt and trace-register-values flags?

matoushybl commented:
Hi, @richardeoin,
thanks for the review! I have updated the README.md with information on using the new features and also I added an entry to the CHANGELOG.md.

Could you please enable workflows for this PR? It seems like they do not run as I am a first time contributor.

Thanks!

richardeoin commented:
Great, thanks for the docs. I haven't figured out how to permanently enable workflows for a particular PR whilst the user is a first-time contributor, instead github seems to need confirmation after every update. In any case this PR looks good now!

bors r+
